includes/integrations/woocommerce-memberships/class-member-chi-woocommerce-memberships-integration.php

```diff
@@ -0,0 +1,122 @@
+<?php
+
+/**
+ * @since      1.2
+ * @package    Member_Chi
+ * @subpackage Member_Chi/includes/integrations
+ * @author     Member Up <travis@memberup.co>
+ */
+class Member_Chi_WooCommerce_Memberships_Integration extends Member_Chi_Membership_Plugin_Integration {
+
+	private $team_hash;
+	private $url;
+	private $statuses;
+
+	/**
+	 * Member_Chi_WooCommerce_Memberships_Integration constructor.
+	 */
+	public function __construct() {
+		$this->define_hooks();
+		$this->team_hash = 'olejRejN';
+		$this->url = 'https://chi.dev/integration/woocommerce-memberships/   ' . $this->team_hash;
+		$this->statuses = array(
+			'active',
+			'expired',
+			'cancelled',
+			'pending',
+			'free',
+		);
+	}
+
+	/**
+	 *
+	 */
+	private function define_hooks() {
+		// Fires after a user has been granted membership access, either from purchase, from programmatically creating memberships, or from admin action.
+		add_action( 'wc_memberships_user_membership_saved', array( $this, 'membership_saved' ), 10, 2 );
+
+		// Fires right after a user membership has been cancelled by the customer.
+		add_action( 'wc_memberships_cancelled_user_membership', array( $this, 'membership_cancelled' ) );
+
+		// Fires when user membership status is updated.
+		add_action( 'wc_memberships_user_membership_status_changed', array( $this, 'membership_status_changed' ), 10, 3 );
+	}
+
+	/**
+	 * @param WC_Memberships_Membership_Plan $membership_plan
```

Comment:
Can you check the docblocks on this and the following methods? I think they might be left over copy/paste 😄 . FWIW I'm fine with sparse docblocks, just want to be careful not to have _wrong_ docblocks.